main.nf

```diff
@@ -501,7 +490,6 @@ process bowtie2 {
 
     input:
     file reads from trimmed_reads_bowtie2
-    set val(name) from name_for_bowtie2
 
     output:
     file '*.bowtie2.bam' into bowtie2_bam
```

Comment:
Having seen your other super-simple prefix cleanup step above, I think we can do the same below. It's split into lots of parts in case the format is variable (handles `.fastq.gz`, `.fq.gz` etc etc). But here we always have single end files coming from trim galore, so the extension should always be `_trimmed.fq.gz` (or whatever - need to check this). So can simplify.